tests/devices/i22/test_fswitch.py

```diff
@@ -49,5 +54,11 @@ def subscription(name, doc):
     )
     assert descriptor_doc["data_keys"] == {"number_of_lenses": expected_data_key}
 
+    assert descriptor_doc["configuration"]["fswitch"]["data"] == {
```

Comment:
Should I be using things like `assert_configuration` here or is this test simple enough to do it like this?

Reply:
Maybe leave it like this, but raise an issue with ophyd-async for making partial `assert_reading` and `assert_configuration`s easier?

Reply:
Done - https://github.com/bluesky/ophyd-async/issues/324